src/Cms/Language.php

```diff
@@ -224,6 +227,9 @@ public static function create(array $props)
 		// update the main languages collection in the app instance
 		App::instance()->languages(false)->append($language->code(), $language);
```

Comment:
We already have `$kirby` and we could clean this up. 

```suggestion
		$kirby->languages(false)->append($language->code(), $language);
```